README.md

```diff
@@ -145,6 +145,15 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Issues**: [Good first issues](https://github.com/search?q=meilisearch+label%3Ahacktoberfest+state%3Aopen+is%3Aissue&type=Issues)
 - **Notes**: Check out [MeiliSearch blogpost](https://blog.meilisearch.com/contribute-hacktoberfest-2021/) for more details
 
+### E
+
+#### **Earthly**
+
+- **Swag**: Stickers (limit 40)
+- **Requirements**: Contribute code to the [Earthly](https://github.com/earthly/earthly) open source build tool or contribute an Earthfile build to any public repo.
+- **How to sign up**: [Grab an issue](https://github.com/earthly/earthly/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest) and complete your PR after accepted please fill out this [form](https://forms.gle/DVkrWCu4yD2wMPmE8).
+- **Notes**: If you need any assistance, [join our slack channel](https://earthly.dev/slack). We are here to help! See our [Blog post](https://earthly.dev/blog/hacktoberfest-2021/) for more details.
+
```

Comment:
Can you move this into proper alphabetical order @biplobsd? 😄 Let's place "E" after "D", please 😆,

Reply:
😃 Sorry for your time!